src/testPane.ts

```diff
@@ -362,9 +368,9 @@ export function updateTestsForEnvironment(
   let jsonData = getEnviroDataFromPython(enviroPath);
 
   if (jsonData) {
-    let nodeID: string;
+    let nodeID: string = "vcast:"
```

Comment:
Is this for a user or for us? What's the standard for other extensions?

Reply:
No one sees this, it is an internal id.
The other extensions that I looked at do not add context menus, so this was a bit "uncharted" so I came up with this really easy fix that leverages the "when clause" regex feature - see package.json.

Reply:
> No one sees this, it is an internal id.
>

Well, we sort of do:

![image](https://github.com/vectorgrp/vector-vscode-vcast/assets/14277540/26eed58a-d991-48de-b73a-954910bbd5e3)

Reply:
Should have said, no one SHOULD see this

Reply:
I did not notice this because my directory name is vcastTutorial :)

Reply:
Fixed